src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const calculator = {
+    add(number) {
+      calculator.result += number;
```

Comment:
use `this`. apply for other operations as well
```suggestion
      this.result += number;

      return this.result;
```